src/Microsoft.AspNetCore.Mvc.TagHelpers/Internal/CryptographyAlgorithms.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Security.Cryptography;
+
+namespace Microsoft.AspNetCore.Mvc.TagHelpers.Internal
+{
+    public static class CryptographyAlgorithms
+    {
+        public static SHA256 CreateSHA256()
+        {
+            SHA256 sha256;
+
+#if NETSTANDARD1_6
+            sha256 = SHA256.Create();
+#else
+            try
+            {
+                sha256 = SHA256.Create();
+            }
+            catch (System.Reflection.TargetInvocationException)
```

Comment:
Should we be looking at the inner exception here? `TIE` is pretty generic. Also, add using.